.ruby-version

```diff
@@ -0,0 +1 @@
+2.2
```

Comment:
From what I understand this file will only be read when changing into the gem's directory (i.e. when developing). Using the latest Ruby in development makes sense, I think.

When the gem is deployed, the `.ruby-version` doesn't matter (if I am not mistaken).